assets/src/js/batch.jsx

```diff
@@ -200,7 +209,7 @@ var App = React.createClass( {
 		}
 
 		// If we are currently processing a batch and there are results.
-		if ( sProcessing.remote_data.current_step !== sProcessing.remote_data.total_steps && sProcessing.remote_data.total_num_results !== 0 ) {
+		if ( sProcessing.remote_data.current_step !== sProcessing.remote_data.total_steps && 0 !== sProcessing.remote_data.total_num_results ) {
```

Comment:
Just noticed this. `sProcessing` what's the `s` for? I am having Hungarian notation flashbacks